monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -221,6 +221,7 @@ class AppComponent extends AuthComponent {
             <Route path={Routes.RegisterPage} render={() => (<RegisterPageComponent onStatusChange={this.updateStatus}/>)}/>
             {this.renderRoute(Routes.LandingPage,
               <SidebarLayoutComponent component={LandingPage}
+                                      sideNavShow={false}
                                       sideNavDisabled={true}
```

Comment:
Do we still need `sideNavDisabled`?

Reply:
could be removed, but hurts no one. I'd say keep it for now, this whole thing will get re-worked in scenarios